.github/workflows/e2e-tests.yml

```diff
@@ -221,136 +216,3 @@ jobs:
         retention-days: 30
         overwrite: true
 
-
-  electron:
```

Comment:
Why did we delete this electron configuration and make the web one above the new electron configuration?

Reply:
They were 99% the same :shrug:

Reply:
> My only concern is that we completely deleted the electron .yml configuration and then update the browser configuration to be electron.

I looked at them both side by side but more reviewers is always welcomed